iamidentityv1/iam_identity_v1_examples_test.go

```diff
@@ -210,6 +220,7 @@ var _ = Describe(`IamIdentityV1 Examples Tests`, func() {
 			// begin-get_api_key
 
 			getAPIKeyOptions := iamIdentityService.NewGetAPIKeyOptions(apikeyID)
+            getAPIKeyOptions.SetIncludeHistory(false)
```

Comment:
Thanks for making this change, and I apologise for being picky. The indentation is off a bit.